plotting-functionality.qmd

```diff
@@ -0,0 +1,43 @@
+---
+title: "Plotting functionality"
+---
+
+Visualisation is key to (data) science and in this section we detail how we approach plotting functionality for our Epiverse packages.
+
+:::{.callout-warning}
+There is ongoing work around theming of plots. Please see [{epiversetheme}](https://epiverse-trace.github.io/epiversetheme/) for more information on this.
+:::
+
+## Approach
+
+Our approach to plotting functionality is twofold:
+
+1. We minimize dependencies used for plotting functions in our packages.
+
+2. We provide detailed plotting examples throughout our vignettes, which may rely on common external dependencies (e.g., {ggplot2}) and include reproducible code.
+
+Epiverse packages may vary from these principles on a case by case basis. Packages that do must document the motivation in the package vignette and [must maintain the standards outlined for dependencies](./dependencies#minimizing-dependencies-and-prioritizing-stability) regardless.
+
+## Reasoning
+
+Plots using {ggplot2} are immensely powerful. Package functions would either need to be highly specific or  complex to manage {ggplot2}'s functionality. We consider this out of scope, if we can provide code for {ggplot2} in vignettes directly.
```

Comment:
Is there one too many spaces before "complex"?

Reply:
```suggestion
Plots using {ggplot2} are immensely powerful. Package functions would either need to be highly specific or complex to manage {ggplot2}'s functionality. We consider this out of scope, if we can provide code for {ggplot2} in vignettes directly.
```